### PR TITLE
Support precision from the commandline

### LINKED ISSUE #42: Encode/decode polyline6 from the cli
Currently there's no option to pass in a precision to polyline.encode or polyline.decode in the CLI, so it's not possible to encode/decode polyline6 geometries from the command line. 

The library already supports variable precision, all that should be needed is to add a command line flag like `--precision 6` to the [binary script](https://github.com/mapbox/polyline/blob/master/bin/polyline.bin.js) and pass the value into the encode and decode functions.

### PR DESCRIPTION
**Pull request by tristen**
I needed this to demo some directions data with polyline6 output.

---

Closes #42

### CODE REVIEW

File: bin/polyline.bin.js [hunk 1]
```diff
@@ -1,43 +1,53 @@
...
-var HELP = 'Provide data from stdin and use with --decode (default), --encode, --toGeoJSON, or --fromGeoJSON\n';
+const HELP = 'Provide data from stdin and use with --decode (default), --encode, --toGeoJSON, or --fromGeoJSON. Optionally provide a precision with --precision=6\n';
 
-var mode = process.argv[2] || '--decode';
-var rawInput = '';
+let precision = 5;
+let rawInput = '';
+const mode = process.argv[2] || '--decode';
+
+if (process.argv[3]) {
```
Comment (elifitch) [inline bin/polyline.bin.js]: Does this mean that we are now enforcing an order in the flags passed to the CLI? Is there a way to make them able to be ordered any way the user wants? If not, we should note the required order in the docs

Comment (tristen) [reply bin/polyline.bin.js]: Yeahhh good point. I waffled with the idea of using https://www.npmjs.com/package/meow but that would be adding a dependency to a package that doesn't have any. But that would be a huge help in making the commandline arguments flexible.

Comment (elifitch) [reply bin/polyline.bin.js]: ooooh this looks like a nice tool. I think that you should feel comfy adding this! Maybe seek comment from the project owner if you feel iffy though

Comment (tristen) [reply bin/polyline.bin.js]: I'm convinced :) going to add it 👍

File: bin/polyline.bin.js [hunk 2]
```diff
@@ -1,47 +1,94 @@
 #!/usr/bin/env node
 
-var polyline = require('../');
+const meow = require('meow');
```
Comment (elifitch) [inline bin/polyline.bin.js]: 😸

**Pull request merged**